Patches/CredentialsPatch.cs

```diff
@@ -214,6 +214,7 @@ public static void Prefix(ModManager __instance)
 
         LateTask.Update(Time.deltaTime);
         CheckMurderPatch.Update();
+        EFC.UpdateUnauthorizedFiles();
```

Comment:
This code will be constantly checked during the game, won't it?

Reply:
I mean I don't want it to affect performance when the mod is constantly reproducing this code

Reply:
All the code is doing is checking if it did find an unauthorized file, checking for the unauthorized files only gets run when someone presses on the play button, so I don't think it should decrease the performance, but if it's a big issue I can move it back to the update on the main menu

Reply:
theres no reason to do this, mods cant be added in runtime lol

Reply:
Please patch MMOnlineManager.Start or something like this

Reply:
Yeah, this will probably be better than a constant check that is checked 20-30 times in one second

Reply:
I'll do something to take down the stress